control-plane/agents/core/src/core/wrapper.rs

```diff
@@ -192,23 +231,40 @@ impl NodeWrapper {
     }
 
     /// Probe the node for liveness
-    pub(crate) async fn liveness_probe(&mut self) -> Result<(), SvcError> {
-        // use the connect timeout for liveness
+    pub(crate) async fn liveness_probe(&mut self) -> Result<Register, SvcError> {
+        //use the connect timeout for liveness
         let timeouts = NodeCommsTimeout::new(
             self.comms_timeouts.connect(),
             self.comms_timeouts.connect(),
             true,
         );
 
-        let mut ctx = self.grpc_client_timeout(timeouts).await?;
-        let _ = ctx
-            .io_engine
-            .get_mayastor_info(rpc::io_engine::Null {})
-            .await
-            .map_err(|_| SvcError::NodeNotOnline {
-                node: self.id().to_owned(),
-            })?;
-        Ok(())
+        let client = self.grpc_client_timeout(timeouts).await?;
+        client.liveness_probe(self.id()).await
+    }
+
+    /// Probe the node for liveness with all known api versions, as on startup its not known
+    /// which api version to reach
+    pub(crate) async fn liveness_probe_all(&mut self) -> Result<Register, SvcError> {
+        //use the connect timeout for liveness
+        let timeouts = NodeCommsTimeout::new(
+            self.comms_timeouts.connect(),
+            self.comms_timeouts.connect(),
+            true,
+        );
+
+        // Set the api version to latest and make a call
+        self.node_state.api_versions = Some(vec![APIVersion::V1]);
+        let client = self.grpc_client_timeout(timeouts.clone()).await?;
+        match client.liveness_probe(self.id()).await {
+            Ok(data) => return Ok(data),
+            Err(_) => debug!("V1 liveness failed on startup, retrying with V0 liveness"),
```

Comment:
hmm wondering this should be an info?
nevertheless, could we improve the log by including the node id? eg:
```suggestion
            Err(_) => debug!(node.id=node_id, "V1 liveness failed on startup, retrying with V0 liveness"),
```